exercises/trinary/trinary_test.rb

```diff
@@ -52,4 +52,28 @@ def test_invalid_trinary_with_digits_is_decimal_0
     skip
```

Comment:
Another extra thing: 
Should this function be called: `test_invalid_trinary_with_invalid_digits_is_decimal_0`